source/includes/inkling-reference/_training-sources.html.md

```diff
@@ -0,0 +1,70 @@
+# Training Sources
+
+The Bonsai Platform supports either `data`, `simulator`, or `generator` as your training source. These support different approaches to training.
```

Comment:
This will change if we try to focus scope on what's actually released as `data` and `generator` are not yet supported.

Also `data` and `generator` really correspond to the same "approach" to training, only `simulator` communicates a different approach. If we keep the features we list here I would instead say:

The bonsai Platform supports training with both real and synthetic data.  The `data`, `simulator` and `generator` keywords are used to describe what kind of training source you would like to use for training.